js/languages/en-US.json

```diff
@@ -55,13 +56,25 @@
     "primaryColor": "Primary Color",
     "secondaryColor": "Secondary Color",
     "textColor": "Text Color",
+    "addresses": {
+      "name": "Name",
+      "company": "Company",
+      "street": "Street",
+      "apt": "Apt/Unit/Suite",
+      "city": "City",
+      "state": "State/Province/Region",
+      "country": "Country",
+      "postalCode": "Postal Code",
```

Comment:
I was just going off of the designs. @morebrownies what do you think of adding in 'Zip' to the label?